docs/imagecreator/quick-start/quick-start-binary.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Quick Start - Binary
```

Comment:
Remove the ` - Binary` since there is no container version of the quicks start guide.